dashboard/src/utils/routeConstants.js

```diff
@@ -1,5 +1,7 @@
-export const AUTH = "auth";
-export const AUTH_LOGIN = "login";
-export const AUTH_SIGNUP = "signup";
+export const HOME = "/dashboard";
+export const AUTH = "dashboard/auth";
+export const AUTH_LOGIN = "dashboard/login";
+export const AUTH_SIGNUP = "dashboard/signup";
 export const USER_PROFILE = "user-profile";
-export const TABLE_OF_CONTENT="toc/:dataset_id"
+export const TABLE_OF_CONTENT = "dashboard/toc/:dataset_id";
+export const OVERVIEW = "overview";
```

Comment:
Why are `USER_PROFILE` and `OVERVIEW` unprefixed, unlike the rest?  (And, why does `HOME` begin with a slash when none of the others do?)

FWIW, you might want to consider constructing the values using a common prefix instead of a literal:
```
const prefix = "dashboard";
export const HOME = "/" + prefix;
export const AUTH = prefix + "/auth";
export const AUTH_LOGIN = prefix + "/login";
export const AUTH_SIGNUP = prefix + "/signup";
export const USER_PROFILE = "user-profile";
export const TABLE_OF_CONTENT = prefix + "/toc/:dataset_id";
export const OVERVIEW = "overview";
```

Reply:
Updated